endpoints/endpoints.go

```diff
@@ -58,12 +58,12 @@ var regionToEndpoint = map[string]map[string]string{
 		"jp-tok":   "https://api.jp-tok.bluemix.net",
 	},
 	"cr": {
-		"us-south": "https://registry.ng.bluemix.net",
-		"us-east":  "https://registry.ng.bluemix.net",
```

Comment:
What is the value for "us-east" if your set the provider to region to "us-east"?

Reply:
@hkantare - IBM Cloud Container Registry doesn't have a regional instance in us-east.
Previously it looks like you were setting this to the same value as the us-south instance.
If you feel that it's important for us-east customers to get invisibly defaulted to the nearest (us-south), then we could set them both to `us.icr.io` , but it seems dangerously opaque to me.